cv_next/app/api/users/fetchUserData/route.ts

```diff
@@ -32,7 +32,7 @@ async function getUserNameHandler(data: { userId: string }) {
       { status: 500 }
     );
   } else {
-    const userName = result.val.username ?? result.val.full_name;
+    const userName = result.val.full_name || result.val.username;
```

Comment:
Wait, is the username here supposed to be display name?
Because if it's username, then we should prefer `username` as default and `full_name` as fallback (well, tbh we shouldn't have a fallback as it always should have a value).
If it's the display name, make sure that it's true throughout the usage of this route.

Reply:
We decide to use the display name for client side username and username as a replacement for user id. You are right the function should be called get display name instead